README.md

```diff
@@ -1,51 +1,80 @@
 | LOGZIO_LISTENER_URL   | Determines protocol, listener host, and port. For example, https://<<LOGZIO_LISTENER_URL>>:8071.Replace <<LOGZIO_LISTENER_URL>> with your region's listener host (for example, listener.logz.io). For more information on finding your account's region, see [Account region](https://docs.logz.io/user-guide/accounts/account-region.html) . |      Yes |
 | LOGZIO_SHIPPING_TOKEN | The [token](https://app.logz.io/#/dashboard/settings/general) of the account you want to ship to.                                                                                                                                                                                                                                             |      Yes |
-| SFCC_HOSTNAME         | Hostname from what host need to send logs (ex. dev01-mysandbox.demandware.net)                                                                                                                                                                                                                                                                |      Yes |
+| SFCC_HOSTNAME         | Hostname from the host that need to send logs from (ex. dev01-mysandbox.demandware.net)                                                                                                                                                                                                                                                       |      Yes |
 | SFCC_CLIENT_ID        | Client id related to the account from where need to send logs. [Learn more](https://documentation.b2c.commercecloud.salesforce.com/DOC3/index.jsp?topic=%2Fcom.demandware.dochelp%2Fcontent%2Fb2c_commerce%2Ftopics%2Faccount_manager%2Fb2c_account_manager_add_api_client_id.html)                                                           |      Yes |
 | SFCC_CLIENT_SECRET    | Client secret related to the account from where need to send logs. [Learn more](https://documentation.b2c.commercecloud.salesforce.com/DOC3/index.jsp?topic=%2Fcom.demandware.dochelp%2Fcontent%2Fb2c_commerce%2Ftopics%2Faccount_manager%2Fb2c_account_manager_add_api_client_id.html)                                                       |      Yes |
+| SFCC_LOG_PATH         | Send logs from site side you need to provide value as `site`, if you need to send only from security path place `security`, if you want to send all log please set up as `all`                                                                                                                                                                |      Yes |
+| AUTO_PARSING          | By default is `false`, for using grok patterns(what was pre defined for you) please set up it as `true`.                                                                                                                                                                                                                                      |       No |
```

Comment:
Please change to : 
**Default:** false
To enable auto parsing (by our prebuilt grok patterns ) set to true.

BTW is the auto parsing is relevant only for operational or security logs as well?